examples/gno.land/p/varmeta/mdui/mdui.gno

```diff
@@ -0,0 +1,364 @@
+	b.elements = append(b.elements, italic)
+	return b
+}
+
+// AddStrikethrough adds strikethrough text to the Builder
+func (b *Builder) AddStrikethrough(text string) *Builder {
+	strikethrough := Strikethrough(text)
+	b.elements = append(b.elements, strikethrough)
+	return b
+}
+
+// AddCodeBlock adds a code block to the Builder
+func (b *Builder) AddCodeBlock(code string) *Builder {
+	codeBlock := CodeBlock(code)
+	b.elements = append(b.elements, codeBlock)
+	return b
+}
+
+// AddAlert adds an alert box with a specified type to the Builder
+func (b *Builder) AddAlert(content, alertType string) *Builder {
+	alert := Alert(content, alertType)
+	b.elements = append(b.elements, alert)
+	return b
+}
+
+// AddCollapsible adds a collapsible section to the Builder
+func (b *Builder) AddCollapsible(title, content string) *Builder {
+	collapsible := Collapsible(title, content)
+	b.elements = append(b.elements, collapsible)
+	return b
+}
+
+// AddFootnote adds a footnote to the Builder
+func (b *Builder) AddFootnote(label, text string) *Builder {
+	footnote := Footnote(label, text)
+	b.elements = append(b.elements, footnote)
+	return b
+}
+
+// AddBadge adds a badge to the Builder
+func (b *Builder) AddBadge(label, color string) *Builder {
+	badge := Badge(label, color)
+	b.elements = append(b.elements, badge)
+	return b
+}
+
+// AddBadgeWithIcon adds a badge with an icon to the Builder
+func (b *Builder) AddBadgeWithIcon(label, color, icon string) *Builder {
+	badgeWithIcon := BadgeWithIcon(label, color, icon)
+	b.elements = append(b.elements, badgeWithIcon)
+	return b
+}
+
+// AddKeyboardShortcut adds a keyboard shortcut to the Builder
+func (b *Builder) AddKeyboardShortcut(keys ...string) *Builder {
+	shortcut := KeyboardShortcut(keys...)
+	b.elements = append(b.elements, shortcut)
+	return b
+}
+
+// AddBlockquoteWithCitation adds a blockquote with an optional citation to the Builder
+func (b *Builder) AddBlockquoteWithCitation(quote, citation string) *Builder {
+	blockquoteWithCitation := BlockquoteWithCitation(quote, citation)
+	b.elements = append(b.elements, blockquoteWithCitation)
+	return b
+}
+
+// AddTableOfContents adds a table of contents to the Builder
+func (b *Builder) AddTableOfContents(headings []string) *Builder {
+	toc := TableOfContents(headings)
+	b.elements = append(b.elements, toc)
+	return b
+}
+
+// Render generates and returns the final Markdown content
+func (b *Builder) Render() string {
+	return strings.Join(b.elements, "\n")
+}
+
+// AddIfElseRender adds conditional content to the Builder based on a condition
+func (b *Builder) AddIfElseRender(condition bool, ifTrue string, ifFalse string) *Builder {
+	result := IfElseRender(condition, ifTrue, ifFalse)
+	b.elements = append(b.elements, result)
+	return b
+}
+
+func (b *Builder) AddBreakLine() *Builder {
+	lineBreak := BreakLine()
+	b.elements = append(b.elements, lineBreak)
+	return b
+}
+
+// BreakLine generates a Markdown line break (two spaces followed by a newline)
```

Comment:
would be better to move below codes into another file, wdyt?

Reply:
Thanks for your suggestion, I were updated the code. Please have a look!